src/en/developer-actions.md

```diff
@@ -0,0 +1,390 @@
+Title: implementing actions in juju charms
+
```

Comment:
can you remember to use your capital letters for the beginning of titles and proper nouns like Juju